### ISSUE TITLE
Remove PairingRDD

### ISSUE THREAD

**Issue by fnothaft** [bigdatagenomics/adam#1122]
After #1108 merges, `PairingRDD` will be unused and should be removed.

**Comment by heuermh** [comment]
Is it still useful, more generally?  https://github.com/bigdatagenomics/utils/issues/16

**Comment by fnothaft** [comment]
IMO, no.

**Comment by heuermh** [comment]
ok, when this issue closes, also close the utils issue linked above as WontFix.

**Comment by fnothaft** [comment]
SGTM!
